docs/faq.rst

```diff
@@ -712,18 +712,28 @@ will make the subsequent rebuilds faster (because it needs to transfer less data
 from the repository). While being faster, the cache needs quite some disk space,
 which might be unwanted.
 
-There is a temporary (but maybe long lived) hack to avoid using lots of disk
-space for chunks.archive.d (see :issue:`235` for details):
+A temporary (but maybe long lived) hack to avoid using lots of disk memory
```

Comment:
AFAIK, the usual term is **disk space**, not **disk memory**.

Reply:
I agree on using "disk space" instead of "disk memory".